__DEFINES/_macros.dm

```diff
@@ -158,7 +158,9 @@
 
 #define iscablecoil(A) istype(A, /obj/item/stack/cable_coil)
 
-#define iscoin(A) is_type_in_list(A, list(/obj/item/weapon/coin, /obj/item/weapon/reagent_containers/food/snacks/chococoin))
+#define iscoin(A) is_type_in_list(A, list(/obj/item/weapon/coin, /obj/item/weapon/reagent_containers/food/snacks/chococoin, /obj/item/weapon/reagent_containers/food/snacks/customizable/candy/coin))
```

Comment:
I think this is no longer needed, see below.

Reply:
Turns out `iscoin()`was already in use by xenoarchaeology in `unknown_trigger_pay2use.dm` So I'll keep it in.